rules/views/app.py

```diff
@@ -403,3 +403,7 @@ def get_context_data(self, **kwargs):
             'rules': rules
         })
         return context
+
+
+class UserEngagementView(AppMixin, TemplateView):
+    template_name = 'rules/user_engagement.html'
```

Comment:
Template name should be `rules/engagement.html`. We have tried to consistently match the name of the template with the url end-point so it is easier to find them.